src/tracking.c

```diff
@@ -290,8 +292,10 @@ void sendTrackingMessage(client *c, char *keyname, size_t keylen, int proto) {
      * in Pub/Sub mode, we can support the feature with RESP 2 as well,
      * by sending Pub/Sub messages in the __redis__:invalidate channel. */
     if (c->resp > 2) {
+        c->flags |= CLIENT_PUSHING;
```

Comment:
don't we need the flag for all the replies in this function (including the one at the very bottom?

Reply:
we do need the flag for all the replies. sorry, i didn't see the full function code before.
i updated it, do you like this way? or should I integrate them into only one `|=` and multiple `&=`

Reply:
i think in the case of tracking.c (specifically a function that's mostly about reply) we can add just one set of `|=` for the entire function, and `&=` in all return statements.
maybe in some, let's consider it on a case-by-case basis, what's cleaner for each one.